### PR TITLE
LPS-191625 Allow create boolean notificationRecipientSettings

### PR DESCRIPTION
**Pull request by seltong**
Related issue: [LPS-191625](https://liferay.atlassian.net/browse/LPS-191625)

### CODE REVIEW

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -6006,6 +6006,7 @@ dynamically-loaded-data=Dynamically Loaded Data
 e=%e
 each-editable-image-element-must-contain-an-img-tag=Each editable image element must contain an {0}IMG{1} tag.
 each-editable-link-element-must-contain-an-a-tag=Each editable link element must contain an {0}A{1} tag.
+each-email-recipient-listed-on-the-to-will-be-sent-separately-and-not-be-able-to-see-others-in-copy=Each email recipient listed on the "To" will be sent separately and not be able to see others in copy.
```
Comment (naraandrade) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:6009]: @seltong I'm thinking we're missing one word in the sentence. Shouldn't it be "...listed on the "To" field" or something similar? Could you confirm with the designer, please?

**Comment by naraandrade** [comment]
Just started reviewing :)

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx [hunk 2]
```diff
@@ -48,6 +51,40 @@ export function EmailNotificationSettings({
...
 			/>
 
+			<ClayForm.Group className="ml-1 row">
+				<div className="mr-2">
+					<ClayCheckbox
+						checked={
+							(values.recipients[0] as EmailRecipients)
+								.singleRecipient
+						}
+						disabled={false}
```
Comment (naraandrade) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx]: @seltong The ClayCheckbox has the property `disabled` as `false` by default, so you don't need to pass it.

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx [hunk 3]
```diff
@@ -162,7 +162,7 @@ export default function EditNotificationTemplate({
 				fromName: {
 					[defaultLanguageId]: '',
 				},
-				singleRecipient: false,
+				singleRecipient: !Liferay.FeatureFlags['LPS-187854'],
```
Comment (naraandrade) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx:165]: @seltong I don't think we should add this property if the FF is disabled. Besides that we would add it with the value `true` which is not expected. You can add the following lines after the initialization of`recipientInitialValue`:
```
if(Liferay.FeatureFlags['LPS-187854']) {
	recipientInitialValue[0] = {...recipientInitialValue[0], singleRecipient: false};
}
```

**Review by naraandrade — CHANGES_REQUESTED** [review]
Hi @seltong, good job :) Just left some comments.

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (seltong) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:6009]: Sure, I will confirm

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx (hunk 2 above)
Comment (seltong) [reply modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/EmailNotificationSettings.tsx]: Yeah, it makes sense. I saw similar cases in some files like [this](https://github.com/liferay/liferay-portal/blob/6ce0b9eb049e99a50285a324a3bd4afd76d340ab/modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/BasicInfoScreen/BasicInfoScreen.tsx#L64-L73) and though that it was a pattern or something like this.
I will fix this in these files too, ok?

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx (hunk 3 above)
Comment (naraandrade) [reply modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx:165]: Oh, I thought the current behavior was the same as when we have the singleRecipient set to false. I'll check the story again.

**Review by murilostodolni — APPROVED** [review]
About back-end: LGTM!

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx (hunk 3 above)
Comment (naraandrade) [reply modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx:165]: We've discussed and agreed that the current behavior is the same as singleRecipient set to true. My bad!

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties (hunk 1 above)
Comment (seltong) [reply modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:6009]: Waiting for reply

**Comment by seltong** [comment]
ci:test:sf

**Review by naraandrade — APPROVED** [review]
@seltong LGTM! I think we can send the change for the language key in another PR if Thiago confirms it's needed.

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by seltong** [comment]
ci:test:sf

**Comment by seltong** [comment]
ci:test:relevant

**Comment by seltong** [comment]
ci:test:object

**Comment by seltong** [comment]
ci:test:object

**Comment by seltong** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
Waiting CI analysis

**Comment by seltong** [comment]
ci:forward:force

**Pull request closed without merging**